README.md

```diff
@@ -36,8 +36,8 @@ jobs:
       - name: GitOps (build, push and deploy a new Docker image)
         uses: Staffbase/gitops-github-action@v4
         with:
-          docker-username: ${{ secrets.DOCKER_USERNAME }}
```

Comment:
as this is a public repo i guess we should keep the variabl generic?

Reply:
Would also leave that as it is.

Reply:
This would make the rollout more difficult for team.